aptos-move/framework/aptos-framework/sources/stake.move

```diff
@@ -197,6 +197,11 @@ module aptos_framework::stake {
         validators: vector<IndividualValidatorPerformance>,
     }
 
+    // The beneficiary address for the operator.
+    struct Beneficiary has key {
```

Comment:
Potentially put this resource on the operator account as well so the contracts don't need to remember to reset it when operator is changed.

Reply:
Done refactoring in this way.